posthog/api/app_metrics.py

```diff
@@ -2,12 +2,40 @@
 
 from rest_framework import mixins, request, response, viewsets
 from rest_framework.decorators import action
+from rest_framework.request import Request
+from rest_framework.response import Response
 
 from posthog.api.routing import StructuredViewSetMixin
 from posthog.models.plugin import PluginConfig
 from posthog.queries.app_metrics.app_metrics import AppMetricsErrorDetailsQuery, AppMetricsErrorsQuery, AppMetricsQuery
 from posthog.queries.app_metrics.historical_exports import historical_export_metrics, historical_exports_activity
-from posthog.queries.app_metrics.serializers import AppMetricsErrorsRequestSerializer, AppMetricsRequestSerializer
+from posthog.queries.app_metrics.serializers import (
+    AppMetricsErrorsRequestSerializer,
+    AppMetricsRequestSerializer,
+    WebhooksMetricsRequestSerializer,
+)
+
+
+class WebhooksViewSet(StructuredViewSetMixin, mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+    def retrieve(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
Ah, the second ID (number) part of the URL is needed because `retrieve` acts on a specific entity, e.g. plugin config with ID 5. To act in aggregate, you should implement the `list` method (there's a default implementation in `mixins.ListModelMixin`, but in this case there's no Django model, so that shouldn't be used).